src/contentsharecontroller/DefaultContentShareController.ts

```diff
@@ -106,6 +110,16 @@ export default class DefaultContentShareController
   }
 
   audioVideoDidStart(): void {
+    this.contentShareTile = this.attendeeAudioVideo.videoTileController.addVideoTile();
+    this.contentShareTile.bindVideoStream(
+      this.contentAudioVideo.configuration.credentials.attendeeId,
+      false,
+      this.mediaStreamBroker.mediaStream,
+      null,
+      null,
```

Comment:
Investigate whether width and height can be derived from MediaStream like it is done here: https://github.com/aws/amazon-chime-sdk-js/blob/0e03fdce362efea84da16af9b28735a72661654a/src/task/CreatePeerConnectionTask.ts#L216

Reply:
Added dimension.